app.R

```diff
@@ -0,0 +1,1267 @@
+# Load Packages ----
+library(shiny)
+library(shinyBS)
+library(shinydashboard)
+library(shinyjs)
+library(shinyWidgets)
+library(boastUtils) 
+
+# Load additional dependencies and setup functions
+source("chisqplot.R") # Loads the script to make the chi sq plot.
+
+# Define UI for App ----
+ui <- list(
+  ## Create the app page ----
+  dashboardPage(
+    skin = "yellow",
+    ### Create the app header ----
+    dashboardHeader(
+      title = "Goodness-of-Fit Test",
+      titleWidth = 250,
+      tags$li(
+        class = "dropdown", 
+        actionLink("info", 
+                   icon("info"))
+        ),
+      tags$li(
+        class = "dropdown",
+        boastUtils::surveyLink(
+          name = "Chi-Square Goodness-fit-Test and Simulation"
```

Comment:
For consistency, would it be better to change this and the overview title to "Chi-Square Goodness-of-Fit-Test and Simulation"